### PR TITLE
[Method References 2] LPS-171231 Convert method reference to ServiceTrackerMap in class MultiLanguageKaleoTaskAssignmentSelector

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I have modified it according to our discussion.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/4558

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:workflow

**Comment by Kvale1733** [comment]
ci:test:stabl;e

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts(6/2 8:25)

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-scripting-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/scripting/internal/util/RulesEngineExecutor.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Jiaxu Wei
+ */
+@Component(enabled = false, service = RulesEngineExecutor.class)
+public class RulesEngineExecutor {
+
+	public Map<String, ?> execute(
```
Comment (tinatian) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-scripting-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/scripting/internal/util/RulesEngineExecutor.java:39]: @W2jx The only usage of this method is also doing merge later, please remove this method.

**Comment by tinatian** [comment]
@W2jx Please see my comment and resend.

Thanks.
Tina

**Pull request closed without merging**